DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Processors/Factory/AnonymizerSettingsFactory.cs

```diff
@@ -0,0 +1,29 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using EnsureThat;
+using Microsoft.Health.Dicom.Anonymizer.Core.Exceptions;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core.Processors.Settings
```

Comment:
Same about the namespaces for classes in this folder.

Reply:
Fixed it.